Pursuant to
https://www.productcompliancespecialists.eu/uploadedFiles/Press_and_Publications/Regulatory_Updates/2013/Regulatory_Update_Mexico_5GHz_Band_3_21-Jan-13.pdf
vacate channels 120, 124 and 128 in the 5GHz band in Argentina regulatory
domain.

See also https://issuetracker.google.com/35527676

--- a/db.txt
+++ b/db.txt
@@ -93,7 +93,9 @@ country AR: DFS-FCC
 	(2402 - 2482 @ 40), (20)
 	(5170 - 5250 @ 80), (17), AUTO-BW
 	(5250 - 5330 @ 80), (24), DFS, AUTO-BW
-	(5490 - 5730 @ 160), (24), DFS
+	# https://www.productcompliancespecialists.eu/uploadedFiles/Press_and_Publications/Regulatory_Updates/2013/Regulatory_Update_Mexico_5GHz_Band_3_21-Jan-13.pdf
+	(5490 - 5600 @ 80), (24), DFS
+	(5650 - 5710 @ 40), (24), DFS
 	(5735 - 5835 @ 80), (30)
 
 country AS: DFS-FCC
